### PR TITLE
DM-6135: vo search simplified panel

### PR DESCRIPTION
**Pull request by ejoliet**
Please, review this branch and let me know. VO search panel will be found in the last right-hand tab on the 'Catalog' search dropdown panel and can be tested by using VO URL value as:  
http://vizier.u-strasbg.fr/viz-bin/votable/-A?-source=J/A+A/402/549
and 'ngc3293' as target.

### CODE REVIEW

File: src/firefly/js/ui/VoSearchPanel.jsx [hunk 1]
```diff
@@ -79,8 +85,27 @@ var sizeArea = () => {
...
 };
 
-function voSearchArea() {
+var voSearchArea = () => {
     return (
-        <div>Search registry box, table result and cone search field here!</div>
+        <ValidationField
+            fieldKey={'vourl'}
+            /*validator={urlValidator}*/
+            tooltip='Enter the VO cone search URL directly (or use the link below to open external NVO search and find the VO cone search URL)'
```
Comment (cwang2016) [inline src/firefly/js/ui/VoSearchPanel.jsx]: please check if this tooltip is shown?

Comment (ejoliet) [inline src/firefly/js/ui/VoSearchPanel.jsx]: You are right, it doesn't. I will fix that. Need to be in a 'initialState' object to be set properly.

**Comment by cwang2016** [comment]
review is done. Looks good.

**Comment by tgoldina** [comment]
Just one more comment: VO URL field should show validation errors. When search is clicked and the VO URL field is invalid or empty, nothing happens. There should be a red validation error mark just like for empty target. (You can set nullAllowed=false and validator in initial state.)

**Pull request merged**